src/palakSJun20/ArrayMissingInteger.java

```diff
@@ -0,0 +1,26 @@
+/*Program 1: Find the missing number in integer array of 1 to 10?
+Hint: You have given an integer array which contains numbers from 1 to 10 but one number
+ is missing, you need to write a Java program to find that missing number in an array.*/
+
+package palakSJun20;
+
+public class ArrayMissingInteger {
+	
+	void findArrayMissingInteger(int[] array){
+		for (int i = 0; i < array.length; i++) {
+			int min = array[0];
+			int max = array[array.length - 1];
+			if (!(array[i] >= min && array[i] <= max)) {
+				System.out.println("Number is missing " + array[i]);
+			}
+		}
+	}
```

Comment:
This logic is incorrect. Output is incorrect.
There is simple and crisp logic available for finding missing number.
The i/p array has to be numbers from 1 to 10 with one number missing - hardcoded. You have taken different array. Please make the corrections.

Reply:
Changes done